cli/src/utils.rs

```diff
@@ -260,6 +261,35 @@ pub fn enable_paint() {
     }
 }
 
+pub trait CommandUtils {
+    fn run(&mut self) -> eyre::Result<Output>;
+    fn get_stdout_lossy(&mut self) -> eyre::Result<String>;
+    fn get_stderr_lossy(&mut self) -> eyre::Result<String>;
+}
```

Comment:
I actually dig this, but let's add a one line doc please.

also can we rename `run` to `exec`?

Reply:
thanks for the suggestion, added the doc and renamed the function

also, i removed the `get_stderr_lossy` function for now, because with it's current logic, it'll always return error and never return stderr content (unless some commands exit with code 0 but print something to stderr?)